modules/fx/arrow-fx-rx2/src/main/kotlin/arrow/fx/rx2/ObservableK.kt

```diff
@@ -23,18 +23,18 @@ class ForObservableK private constructor() {
   companion object
 }
 typealias ObservableKOf<A> = arrow.Kind<ForObservableK, A>
-typealias ObservableKKindedJ<A> = io.kindedj.Hk<ForObservableK, A>
 
 @Suppress("UNCHECKED_CAST", "NOTHING_TO_INLINE")
 inline fun <A> ObservableKOf<A>.fix(): ObservableK<A> =
   this as ObservableK<A>
 
 fun <A> Observable<A>.k(): ObservableK<A> = ObservableK(this)
 
+@Suppress("UNCHECKED_CAST")
 fun <A> ObservableKOf<A>.value(): Observable<A> =
-  fix().observable
+  fix().observable as Observable<A>
```

Comment:
Why is this necesssary? What's the complaint?

Reply:
<img width="683" alt="Screenshot 2019-09-24 at 22 23 48" src="https://user-images.githubusercontent.com/437967/65547646-2dad8100-df1a-11e9-934b-ac82a9b67b07.png">
It seems it cannot auto cast `Observable<out A>` to `Observable<A>`

Reply:
see if there's any of the @annotations that may fix it :D

Reply:
where should I look for annotations? I checked here: https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-annotation.html, but I didn't find any thing useful.

Reply:
JvmSuppressWildcards or JvmWildcard

Reply:
Both of them dont' work :(

Reply:
do I need to do anything to fix this? @pakoito

Reply:
No, leave it as is.

I'd suggest testing a bit just in case.